src/native/monodroid/debug-constants.cc

```diff
@@ -0,0 +1,10 @@
+#include "debug.hh"
+#include "shared-constants.hh"
+
+using namespace xamarin::android;
+using namespace xamarin::android::internal;
+
+extern "C" const char *__get_debug_mono_log_property (void)
```

Comment:
Pity I didn't notice this in e99b02b788, but *generally* double-underscore `__` is reserved for the compiler + toolchain, and should not be used in "user" code (that's us! user code!).

I also don't understand why `DEBUG_MONO_LOG_PROPERTY` is "special" here in getting a `__get_` function + macro, vs. e.g. `DEBUG_MONO_ENV_PROPERTY` which is a `static inline constexpr`.

Reply:
Yeah, we shouldn't use `__` in macros, functions, classes etc, but it's "legacy" so I opted not to touch it.  As for the logic behind it? I have no idea, it's another "legacy" thing - I assume that this code was called by something else (either managed or native, likely the former) so I felt it safer to leave it alone.  And the macro calling this function is to have one location where the name is fetched, "just in case".